Another rake osx:deps:install problem in osx 10.7.1

I'm using 1.9.2 via rvm and I get this:

Jeremys-MacBook-Pro:shoes jeremy$ rake osx:deps:install --trace
rake aborted!
no such file to load -- cucumber
/Users/jeremy/.rvm/rubies/ruby-1.9.2-p290/lib/ruby/site_ruby/1.9.1/rubygems/custom_require.rb:36:in `require'
/Users/jeremy/.rvm/rubies/ruby-1.9.2-p290/lib/ruby/site_ruby/1.9.1/rubygems/custom_require.rb:36:in`require'
/Users/jeremy/Projects/shoes/shoes/Rakefile:16:in `<top (required)>'
/Users/jeremy/.rvm/gems/ruby-1.9.2-p290@global/gems/rake-0.9.2/lib/rake/rake_module.rb:25:in`load'
/Users/jeremy/.rvm/gems/ruby-1.9.2-p290@global/gems/rake-0.9.2/lib/rake/rake_module.rb:25:in `load_rakefile'
/Users/jeremy/.rvm/gems/ruby-1.9.2-p290@global/gems/rake-0.9.2/lib/rake/application.rb:495:in`raw_load_rakefile'
/Users/jeremy/.rvm/gems/ruby-1.9.2-p290@global/gems/rake-0.9.2/lib/rake/application.rb:78:in `block in load_rakefile'
/Users/jeremy/.rvm/gems/ruby-1.9.2-p290@global/gems/rake-0.9.2/lib/rake/application.rb:129:in`standard_exception_handling'
/Users/jeremy/.rvm/gems/ruby-1.9.2-p290@global/gems/rake-0.9.2/lib/rake/application.rb:77:in `load_rakefile'
/Users/jeremy/.rvm/gems/ruby-1.9.2-p290@global/gems/rake-0.9.2/lib/rake/application.rb:61:in`block in run'
/Users/jeremy/.rvm/gems/ruby-1.9.2-p290@global/gems/rake-0.9.2/lib/rake/application.rb:129:in `standard_exception_handling'
/Users/jeremy/.rvm/gems/ruby-1.9.2-p290@global/gems/rake-0.9.2/lib/rake/application.rb:59:in`run'
/Users/jeremy/.rvm/gems/ruby-1.9.2-p290@global/gems/rake-0.9.2/bin/rake:32:in `<top (required)>'
/Users/jeremy/.rvm/gems/ruby-1.9.2-p290@global/bin/rake:19:in`load'
/Users/jeremy/.rvm/gems/ruby-1.9.2-p290@global/bin/rake:19:in `<main>'

Do I need to install cucumber via brew?

thats the problem, rake osx:reps:install is failing on cairo:

ld: lto: could not merge in .libs/cairo-analysis-surface.o because Unknown instruction for architecture x86_64
collect2: ld returned 1 exit status

I guess the default cairo on homebrew fails with llvm-gcc?

--- a/make/darwin/env.rb
+++ b/make/darwin/env.rb
@@ -23,7 +23,7 @@ if pkg_config and pkgs.include?("cairo") and pkgs.include?("pango")
   PANGO_LIB = ENV['PANGO_LIB'] ? "-L#{ENV['PANGO_LIB']}" : `pkg-config --libs pango`.strip
 else
   # Hack for when pkg-config is not yet installed
-  CAIRO_CFLAGS, CAIRO_LIB, PANGO_CFLAGS, PANGO_LIB = "", "", "", "" if `which pkg-config` == ""
+  CAIRO_CFLAGS, CAIRO_LIB, PANGO_CFLAGS, PANGO_LIB = "", "", "", ""
 end
 png_lib = 'png'
 

